lib/nitcorn/examples/src/htcpcp_server.nit

```diff
@@ -0,0 +1,67 @@
+import nitcorn
+
+class HTCPCPAction
+  super Action
+  var brewing = false
+
+  redef fun answer(http_request, turi) do
```

Comment:
Indenting in Nit is usually a tab character, can you replace the two-spaces by a tab character ?